docs/integrating-with-unity/defining-actions.mdx

```diff
@@ -10,20 +10,20 @@ import TabItem from '@theme/TabItem';
 This section refers to an "action" as any instruction that a bot can issue to its GameObject.
 This can be anything from movement to casting abilities to navigating UI components.
 
-It is common for actions to interact with entities in the [game state](./defining-entities-and-their-states) and their corresponding GameObjects.
-The [`RGFindUtils`](../creating-bots/csharp/utilities#rgfindutils) class contains utility functions for finding such GameObjects within the scene and is
-useful for these types of actions.
+It is common for actions to interact with entities in the [game state](./defining-states) and their corresponding GameObjects.
```

Comment:
Is RGFindUtils going away as well?

Reply:
`RGFindUtils` still exists, but isn't necessary or useful to the developer in the context of actions.  Only the click button action uses it anymore and that is all hidden internally in a custom action implementation.

Regular actions work now because we dynamically add an `RGActionHandler` Behaviour at runtime to any GameObject that has any Behaviour(s) with [RGAction] attributes.  The action is then invoked against that actionhandler which has mappings of action names to Delegates of how to invoke that action correctly on the correct Behaviour on that same gameObject.

TLDR;  Unless you're doing something outside the box like trying to call an action on a gameobject not controlled by your bot client... The action handling is now all efficiently handled through the actionhandler, avoiding having to find objects on every action call.

Reply:
I see, very cool!